styles/style.css

```diff
@@ -121,7 +149,10 @@ p {
 
 .hero {
   background-image: url("header-bike.jpg");
+  background-image: url(../images/header-bike.jpg);
   background-color: var(--grey-light);
+  padding: 80px 30px;
```

Comment:
where possible, try to use relative units (em, rem, %) rather than px - will make styling more consistent across the site